middleware/ordered_group_test.go

```diff
@@ -181,18 +181,17 @@ func TestOrderedIDsGetOrder(t *testing.T) {
 func TestOrderedIDsSlots(t *testing.T) {
 	o := newOrderedIDs()
 
-	noError(t, o.AddSlot("first", After))
-	noError(t, o.AddSlot("second", After))
+	noError(t, o.AddSlot(After, "first", "second"))
 	noError(t, o.Add(mockIder("second"), After))
-	noError(t, o.InsertSlot("fourth", "second", After))
+	noError(t, o.InsertSlot("second", After, "fourth"))
 	noError(t, o.Insert(mockIder("third"), "second", After))
 	noError(t, o.Add(mockIder("fifth"), After))
 
 	if e, a := []string{"first", "second", "third", "fourth", "fifth"}, o.List(); !reflect.DeepEqual(e, a) {
 		t.Errorf("expect %v order, got %v", e, a)
 	}
 
-	if o.AddSlot("first", After) == nil {
+	if o.AddSlot(After, "first") == nil {
 		t.Error("expect error, got nil")
 	}
 
```

Comment:
Change looks good, but need test cases to ensure correct behavior if the `ids...` variadic list is empty.